var/spack/repos/builtin/packages/python/package.py

```diff
@@ -225,7 +225,8 @@ class Python(AutotoolsPackage):
 
     # Ensure that distutils chooses correct compiler option for RPATH on fj:
```

Comment:
Has this patch been submitted upstream yet? The distutils module is deprecated in Python 3.10 and will be removed in Python 3.12, so it might be better to patch setuptools or sysconfig instead.

Reply:
The distutil related issue seems to closed.
https://bugs.python.org/issue25229

Reply:
Yes, that's because distutils is deprecated. Now that we build everything with pip, and now that pip replaces distutils with setuptools automatically, and now that setuptools vendors all the functionality of distutils, I'm not sure if your patch is even being used. I would suggest trying to submit a patch to setuptools instead.

Reply:
The patch is not required for setuptools.
In setuptools, is_gcc is used only hp-ux, and all compilers on linux use -Wl,-R for specify runtime library directory.